x-pack/plugins/cloud_security_posture/server/routes/benchmark_rules/find/find.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { transformError } from '@kbn/securitysolution-es-utils';
+import {
+  FindCspBenchmarkRuleRequest,
+  FindCspBenchmarkRuleResponse,
+  findCspBenchmarkRuleRequestSchema,
+} from '../../../../common/types/latest';
```

Comment:
better to explicitly use v1 here instead
think of the use case of adding a newer version

Reply:
https://github.com/elastic/kibana/pull/173066/files#diff-75f8ea25f299cb586a1a72930204fefeed06db979233f267c60b0a48fa8a96efR12